src/GuestConfiguration/GuestConfiguration/Az.GuestConfiguration.psd1

```diff
@@ -107,7 +107,7 @@ PrivateData = @{
         ReleaseNotes = 'Initial Release'
 
         # Prerelease string of this module
-        Prerelease = 'preview'
+        #Prerelease = 'preview'
```

Comment:
This should still be a preview release. Please revert. We agreed that it would not GA until if it determined to be a separate module or not.

Reply:
I was incorrect in this statement.

Reply:
As per our conversation with Scott, there won't be a "*-preview" in the version in the end. but it should be less than 1.0.0. We've already released 0.9.0-preview. So. it should be greater that without "preview" suffix. Once we determine a location and release it from there, it should be released as 1.0.0. This module is not preview for all practical purposes. It is like a standalone stable/GA module.